layouts/index.html

```diff
@@ -16,13 +33,27 @@ <h2>Charm Engineering Releases</h2>
             </p>
             <p class="u-no-max-width">
               For a release to show here, it must be part of one of the specified Github Teams or
-              Launchpad project groups. Github projects must have at least one Github Release,
-              Launchpad projects are scraped for tags. The latest three releases are shown for each
+              Launchpad project groups. Github projects must not be Archived, Launchpad projects
+              are scraped for tags. The latest three releases are shown for each
               repository.
             </p>
```

Comment:
I wonder if this para should also move into the accordion while we're tidying this up?

Reply:
Good point, I would agree it belongs there!